service/jmx/src/main/java/org/jolokia/service/jmx/handler/ExecHandler.java

```diff
@@ -187,9 +195,10 @@ private OperationAndParamType extractOperationTypes(MBeanServerConnection pServe
                 // Number of arguments dont match
                 continue OUTER;
             }
-            for (int i=0;i<infos.length;i++) {
+            for (int i = 0; i < infos.length; i++) {
                 String type = infos[i].getType();
-                if (!type.equals(pTypes.get(i))) {
+                // Object is here for an eventual case of type erasure
+                if (!type.equals(pTypes.get(i)) && !type.equals(Object.class.getName())) {
```

Comment:
Do you have an example where type erasure cause `MBeanParameterInfo.getType()` to return `Object` ?